### DISCUSSION TITLE
how to run forever?

### DISCUSSION THREAD

**Discussion by fre2mansur** [albertobsd/keyhunt#333] [General]
Im running on cloud Linux, how to run forever? and get notified once you get hit? instead of watching the terminal all the time?

**Comment by albertobsd** [comment]
That can be done with another script external to keyhunt, you need to detect when the KEYFOUND file is created and it need to notify you somehow...

**Reply by fre2mansur** [reply]
keyfound file comes with this repository should I delete the file and wait for a new keyfound file to be created? 
sorry I'm new here.

**Reply by sssergy** [reply]
Yes, you can delete this file.
You can then track its presence, since when the private key is found, it will be re-created.
